#Header
std_msgs/Header header

#Data
std_msgs/Int32 headlightState
std_msgs/Int32 wiperState
std_msgs/Int32 sirenState
std_msgs/Int32 turnsignalState
